### PR TITLE
Add contact-form

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
#Mobile-Version

Added contact-form 👍🏽

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -179,5 +179,37 @@ <h3>Multi-Post Stories</h3>
...
+<form class="contact-form" action="https://formspree.io/f/mknyzlre" method="post">
+<input type="text" name="username" placeholder="Text Input" maxlength="30" required>
+<input type="email" name="email" placeholder="jeremy@gmail.com" maxlength="30" required>
+<textarea name="text" placeholder="Enter text here" maxlength="500" required>
+</textarea>
+
+<div class="btn">
+  <button type="submit">See Project</button>
+</div>
+</form>
```
Comment (mateo951) [inline index.html:196]: Your form is working like a charm 💪😎 Well done

Comment (mateo951) [reply index.html:196]: HTML validations are perfectly setup ✔

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

📝👌 **Your project highlights** 🔥😎 
- Linters are on point ✔
- Great Readme file 🙌
- GitHub Flow is applied perfectly ✔

Your project is complete!  There is nothing else to say other than... it's time to merge it 🔥 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (kinginthenorthcodez) [reply index.html:196]: Thanks for the review 👍🏽

**Pull request merged**